internal/msalbase/oauth_response_base_test.go

```diff
@@ -9,20 +9,20 @@ import (
 )
 
 var oauthResponse = `{}`
-var oauthResponseWithError = `{"error" : "invalid request", "error_codes" : [300]}`
+var oauthResponseWithError = `{"error" : "invalid request", "error_description": "missing payload content", "error_codes" : [300]}`
 
 func TestCreateOAuthResponseBase(t *testing.T) {
-	_, err := CreateOAuthResponseBase(404, oauthResponse)
+	_, err := CreateOAuthResponseBase(404, []byte(oauthResponse))
 	actualError := errors.New("HTTP 404")
 	if err.Error() != actualError.Error() {
 		t.Errorf("Actual error %v differs from expected error %v", err, actualError)
 	}
-	expectedErr := errors.New("invalid request")
-	_, err = CreateOAuthResponseBase(300, oauthResponseWithError)
+	expectedErr := errors.New("invalid request: missing payload content")
```

Comment:
These expectedErr are brittle.  Any change to the text of the error can cause the tests to fail.

I know that wasn't your test, but can we change this so we just test the given inputs against outputs?  The output can be simply tested to see if it has an error, not a specific error.  This is sufficient 99.9% of the time.

Reply:
Agreed, fixed.